.travis.yml

```diff
@@ -87,6 +87,12 @@ script:
         && [ "${TRAVIS_SECURE_ENV_VARS}" != "false" ] \
         && [ "${TRAVIS_PULL_REQUEST_BRANCH}" != "" ];
     then
+      # Skip if webapp isn't modified.
+      git diff --name-only FETCH_HEAD...$TRAVIS_BRANCH | grep ^webapp/ || {
```

Comment:
nit: Consider `"FETCH_HEAD...${TRAVIS_BRANCH}"`. The `${}` is consistent with surrounding code and the quoting will deal with branches that contain funky characters (like spaces) in the branch name.